libvast/vast/uuid.hpp

```diff
@@ -13,25 +13,24 @@
 
 #pragma once
 
-#include <array>
+#include "vast/byte.hpp"
+#include "vast/detail/operators.hpp"
+#include "vast/span.hpp"
+#include "vast/uuid.hpp"
```

Comment:
`operators.hpp` seems unused, and it probably also should not include itself.

Reply:
Actually, `detail::totally_orderd<T>` is used here. I've removed the reflexive include.